### ISSUE TITLE
[A11y] Snack bar cannot be dismissed if there is another action.

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#87215]
b/194869827

## Steps to Reproduce

```dart
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Material App',
      theme: ThemeData.dark(),
      home: Home(),
    );
  }
}

class Home extends StatefulWidget {
  @override
  State<Home> createState() => _HomeState();
}

class _HomeState extends State<Home> {

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Material App Bar'),
      ),
      body: Builder(
        builder: (BuildContext context) {
          return RaisedButton(
            child: Text('button'),
            onPressed: () {
              final snackBar = SnackBar(
                content: Text('Yay! A SnackBar!'),
                action: SnackBarAction(
                  label: 'Undo',
                  onPressed: () {
                    // Some code to undo the change.

                  },
                ),
              );
              ScaffoldMessenger.of(context).showSnackBar(snackBar);
            },
          );
        },
      ),
    );
  }
}
```
1. turn on talkback
2. tap the button to show the snackbar

The snackbar cannot be dismissed unless perform the action.

According to the guideline:

Snackbars that present a user action, such as retry or undo, do not time out until the user performs another action in the application or if the user explicitly dismisses it.

There doesn't seem to have a way to explicitly dismisses it.

**Comment by chunhtai** [comment]
update: looks like WAI. you can dismiss it through local context menu, and it matches native behavior
